### PR TITLE
[parity-crypto] Release 0.6.1

### PR DESCRIPTION
**Pull request by dvdplm**
…and fix construction of `ZeroesAllowedMessage`

### CODE REVIEW

File: parity-crypto/src/publickey/mod.rs [hunk 1]
```diff
@@ -44,7 +44,7 @@ use secp256k1::ThirtyTwoByteHash;
 /// converted to a `[u8; 32]` which in turn can be cast to a
 /// `secp256k1::Message` by the `ThirtyTwoByteHash` and satisfy the API for
 /// `recover()`.
-pub struct ZeroesAllowedMessage(H256);
+pub struct ZeroesAllowedMessage(pub H256);
```
Comment (dvdplm) [inline parity-crypto/src/publickey/mod.rs:47]: Forgot this. :/

Comment (ordian) [reply parity-crypto/src/publickey/mod.rs:47]: An alternative would be to implement `From<H256> for ZeroesAllowedMessage`, but this is also fine

File: parity-crypto/CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog].
 
 ## [Unreleased]
 
+## [0.6.1] - 2020-04-11
+- Add `recover_allowing_all_zero_message()` and `ZeroesAllowedMessage` to accomodate ethereum's `ecrecover` builtin. [#369](https://github.com/paritytech/parity-common/pull/369)
```
Comment (ordian) [inline parity-crypto/CHANGELOG.md:10]: wrong link :)

**Comment by dvdplm** [comment]
Published 0.6.1

**Pull request merged**